bundles/config/org.eclipse.smarthome.config.discovery.test/src/test/groovy/org/eclipse/smarthome/config/setup/test/inbox/DynamicThingUpdateOSGITest.groovy

```diff
@@ -44,12 +44,13 @@ import org.junit.Test
  * A dummy {@link ThingHandler} and {@link ThingHandlerFactory} is used to detect an updated event.
  * 
  * @author Michael Grammling - Initial Contribution
+ * @author Thomas Höfer - Added representation
  */
 class DynamicThingUpdateOSGITest extends OSGiTest {
 
     def DEFAULT_TTL = 60
 
-        final BINDING_ID = 'dnamicUpdateBindingId'
+    final BINDING_ID = 'dnamicUpdateBindingId'
```

Comment:
typo (I see that it was not you, but please fix it anyhow :-)): should be "dynamic"